src/devtools/client/webconsole/components/Output/Message.js

```diff
@@ -181,6 +182,7 @@ class Message extends Component {
       );
 
       this.onViewSourceInDebugger({ ...frame, url: frame.source });
+      dismissNag(Nag.FIRST_CONSOLE_NAVIGATE);
```

Comment:
hmm - the nag didnt go away for me when i clicked fast forward. I think this needs to be in the fastforward click handler...